packages/quantic/cypress/integration/example-search.cypress.ts

```diff
@@ -1,42 +1,63 @@
 import {
+  selectors as search,
   searchFor,
   selectResultPage,
   setupAliases,
   sortByDateDescending,
+  selectRegularFacetValue,
 } from '../page-objects/example-search';
 
 describe('example-search', () => {
-  const exampleSearchUrl = 'http://localhost:3334/preview/c/exampleSearch';
+  const pageUrl = `${Cypress.env('examplesUrl')}/s/full-search-example`;
+
+  Cypress.on('uncaught:exception', (err) => {
+    if (
+      err.message.includes(
+        'navigator.sendBeacon is not supported in this browser'
```

Comment:
This is to suppress the error? Why is this needed? Might be worth adding a comment to explain.

Reply:
@tedre191 The error comes from [Coveo.analytics.js](https://github.com/coveo/coveo.analytics.js/blob/master/src/client/analyticsBeaconClient.ts#L21). Basically, it happens because the browser when the browser switches between tests, the UA event buffer might not have been emptied. So, when Coveo.analytics.js tries to flush the events, `navigator.sendBeacon` does not exist anymore.

Now, I think we can safely ignore it because, either the test does not assert UA requests, or it will wait for UA requests to be made (preventing this error at the same time).